### DISCUSSION TITLE
Quarto Blog Error- Cannot find file specified: execute-results

### DISCUSSION THREAD

**Discussion by KalePenguin** [quarto-dev/quarto-cli#6143] [Q&A] [answered]
### Description

Attempting to render my quarto blog index file and everything is fine until the very end. Then I get this message:

output file: index.knit.md

ERROR: NotFound: The system cannot find the file specified. (os error 2), remove 'G:\My Drive\R codes\My Last Chance\Last Chance\posts\July 2023 Update\index_files\execute-results'

NotFound: The system cannot find the file specified. (os error 2), remove 'G:\My Drive\R codes\My Last Chance\Last Chance\posts\July 2023 Update\index_files\execute-results'
    at Object.removeSync (deno:runtime/js/30_fs.js:157:9)
    at removeIfExists (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:5681:14)
    at removeFreezeResults (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:86411:5)
    at renderExecute (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:86499:9)
    at async renderFiles (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:86591:51)
    at async renderProject (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:86797:25)
    at async Command.fn (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:103712:46)
    at async Command.execute (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:8437:13)
    at async quarto (file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:127539:5)
    at async file:///C:/PROGRA~1/RStudio/RESOUR~1/app/bin/quarto/bin/quarto.js:127557:9

G:\My Drive\R codes\My Last Chance\Last Chance>quarto check

[>] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.1: OK
      Dart Sass version 1.55.0: OK
[>] Checking versions of quarto dependencies......OK
[>] Checking Quarto installation......OK
      Version: 1.3.353
      Path: C:\Program Files\RStudio\resources\app\bin\quarto\bin
      CodePage: 1252

[>] Checking basic markdown render....OK

[>] Checking Python 3 installation....(None)

      Unable to locate an installed version of Python 3.
      Install Python 3 from https://www.python.org/downloads/

[>] Checking R installation...........OK
      Version: 4.2.0
      Path: C:/PROGRA~1/R/R-42~1.0
      LibPaths:
        - C:/Users/kylep/AppData/Local/R/win-library/4.2
        - C:/Program Files/R/R-4.2.0/library
      knitr: 1.43
      rmarkdown: 2.22

[>] Checking Knitr engine render......OK
I have attempted to add in the "execute-results" folder manually, but the render deletes it. This is the first time running into this issue since the most recent release of ["Mountain Hydrangea" Release (583b465e, 2023-06-05) for windows].

**Comment by cscheid** [comment]
`G:\My Drive` I believe this is a google drive? We don't recommend you run quarto from a network drive, because of filesystem synchronization issues. Can you try to run in a filesystem location that's local to your computer and see if the issue persists?

**Reply by KalePenguin** [reply]
Thank you. I had already tried to set the working directory to another source. Based on your suggestion, I copied the entire contents of the project folder into a new project on my local drive. The file was then rendered without issue. I will be sure to keep everything on the local drive in the future. Thank you for the help. I am a bit of a quarto newbie.

**Reply by asegun-cod** [reply]
I moved my repo from google drive (a network drive) to my C drive (document folder) and this solved the problem for me.

**Reply by issactoast** [reply]
I ran into this issue today after I update quarto to 1.4.xx. I haven't had issue with 1.2.xx quarto and I have been rendering on Google drive without any issues.

* Now I found that sync issue happens when I try to set format as pdf

**Reply by mcanouil** [reply]
Unfortunately, rendering to a sync drive is known to have issue.
For now, it's recommended to move the files after rendering.

In general, it's not a good thing to work on a sync drive when you generate multiple temporary files, because syncing can make the file non readable/editable which can cause various issues.

**Reply by cscheid** [reply]
(Just to be clear, we're aware that this is a _Quarto bug_. It's just that we appear to be inheriting this bug from the programming language we use, and we haven't been able to isolate precisely what's happening.)

**Reply by mcanouil** [reply, marked as answer]
FYI:
- https://github.com/quarto-dev/quarto-cli/issues/8239
- https://github.com/quarto-dev/quarto-cli/labels/file-systems

**Reply by issactoast** [reply]
Okay, after I play around the different version, quarto 1.2.x doesn't have this issue, but after 1.3.x similar sync error occur. ~~Since there is an #8239, I will write up my situation there.~~ I write up an new issue using the template here; #8457
